src/site/layouts/vet_center.drupal.liquid

```diff
@@ -26,6 +26,66 @@
+                                         href="https://maps.google.com?saddr=Current+Location&daddr={{ fieldAddress.addressLine1 }}, {{ fieldAddress.locality }}, {{ fieldAddress.postalCode }}"
+                                         target="_blank" rel="noopener noreferrer">Directions
+                                        </a>
+                                    </div>
+                                </div>
+                                <h3 class="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">Direct line</h3>
+                                <div class="vads-u-margin-bottom--3">
+                                    <div class="main-phone vads-u-margin-bottom--1">
+                                        </strong><a href="tel:{{ fieldPhoneNumber }}">{{ fieldPhoneNumber }}</a>
+                                    </div>
+                                </div>
+                               <div class="vads-u-margin-bottom--3">
+                                   <h3 class="vads-u-font-size--lg vads-u-margin-top--0 vads-u-line-height--1 vads-u-margin-bottom--1">Other locations</h3>
+                                   <p>Vet Centers are community based to be more <br>
+                                       accessible in areas where you live.<br>
+                                       <a href="{{ entityUrl.path }}/locations">
+                                           View more {{ entityLabel }} locations
+                                       </a>
+                                   </p>
+                               </div>
+                              {% include 'src/site/includes/vet_center_hours.liquid' with vetCenterHours = fieldOfficeHours %}
+                          </section>
+                      </div>
+                  </div>
+              </div>
+
+              <div class="usa-width-one-third inline-table-helper vads-u-order--first small-screen:vads-u-order--initial
+                vads-u-margin-bottom--2 vads-u-margin-left--auto facility">
+                  <!-- facility image -->
+                    {% if fieldMedia.entity.image != empty %}
+                        {% assign image = fieldMedia.entity.image %}
+                        <a href="#" rel="noopener noreferrer" target="_blank"
+                           id="google-map-link-image">
+                            <img class="facility-img" src="{{ image.derivative.url }}"
+                                 alt="{{ image.alt }}" title="{{ image.title }}"
+                                 width="{{ image.derivative.width }}"
+                                 height="{{ image.derivative.height }}" />
+                        </a>
+                    {% endif %}
+                  <!-- facility map -->
+                  <div data-widget-type="facility-map"
+                       data-facility="{{ fieldFacilityLocatorApiId }}">
+                  </div>
+              </div>
```

Comment:
Lines 69:86 are an exact duplicate of [this](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/health_care_local_facility.drupal.liquid#L114-L130). Can we refactor?